test/ketu/async/sink_test.clj

```diff
@@ -71,6 +71,8 @@
           sink (sink/sink ch opts)]
       (async/>!! ch "v")
       (async/close! ch)
+      ;; Allow the worker thread to consume all messages before aborting input
+      (Thread/sleep 500)
```

Comment:
Shouldn't be needed if `sink/stop!` keeps the original order of invocations.

Reply:
Indeed. Removed.